unity-renderer/Assets/Scripts/MainScripts/DCL/FeatureFlags/Data/FeatureFlag.cs

```diff
@@ -7,6 +7,8 @@ public class FeatureFlag
 {
     public readonly Dictionary<string, bool> flags  = new Dictionary<string, bool>();
     public readonly Dictionary<string, FeatureFlagVariant> variants  = new Dictionary<string, FeatureFlagVariant>();
+    
+    public const string GPU_ONLY_MESHES = "ab-use-gpu-only-meshes";
```

Comment:
Done! Duplicated the flag name on GLTF and AB use. Seemed cleaner than adding unnecessary dependencies to a Util class. Especially because flag name is not going to change